tweb/src/main/frontend/src/js/vue/components/ProjectCard.vue

```diff
@@ -0,0 +1,187 @@
+<template>
+	<div class="project-container">
+		<div class="project-icon">
+			<img v-bind:src="project.logoLocation">
+		</div>
+		<a v-bind:href="project.projectLink">
+			<div class="project-request-details" :style="{ 'background': project.overviewColor }">
+				<div class="project-request-name"><h3>{{ project.name }}</h3></div>
+				<div class="project-request-description"><p>{{ project.description }}</p></div>
+				<hr />
+				<div class="requestCountContainer">
+					<div class="requestNumber"><p>#{{ boundedRequests }}</p></div>
+					<div class="requestLabel"><p>active</p><p>requests</p></div>
+				</div>
+			</div>
+		</a>
+	</div>
+</template>
+
+<script lang="ts">
+    import {Component, Prop, Vue} from "vue-property-decorator";
+	import ProjectOverviewDetail from "../models/ProjectOverviewDetail"
+
+    @Component
+    export default class ProjectCard extends Vue {
+
+		defaultProject: ProjectOverviewDetail = {
+			name: "FundRequest",
+			description: "Marketplace for Software Development",
+			overviewColor: "#000000",
+			projectLink: "https://fundrequest.io/requests?",
+			logoLocation: "/assets/img/logo-single180.png",
+			activeRequests: 5
+		};
+
+		@Prop({ 
```

Comment:
A possible solution is to define the default object outside the class (this is ok, since it's contained in the .vue container). And create a copy of object as default value.
```
const defaultProject: ProjectOverviewDetail = Object.assign(new ProjectOverviewDetail(), {
    name: "FundRequest",
    description: "Marketplace for Software Development",
    overviewColor: "#000000",
    projectLink: "https://fundrequest.io/requests",
    logoLocation: "/assets/img/logo-single180.png",
    activeRequests: 5
});

@Component
export default class ProjectCard extends Vue {
    @Prop({default: {...defaultProject}}) project: ProjectOverviewDetail;
```

But since you merge the objects, it's also possible to leave out the default in the prop (just add a `v-if="project"` to the html, so it will give no errors when the object is null.

Reply:
A possible solution is to define the default object outside the class (this is ok, since it's contained in the .vue container). And create a copy of object as default value.
```
const defaultProject: ProjectOverviewDetail = Object.assign(new ProjectOverviewDetail(), {
    name: "FundRequest",
    description: "Marketplace for Software Development",
    overviewColor: "#000000",
    projectLink: "https://fundrequest.io/requests",
    logoLocation: "/assets/img/logo-single180.png",
    activeRequests: 5
});

@Component
export default class ProjectCard extends Vue {
    @Prop({default: {...defaultProject}}) project: ProjectOverviewDetail;
```

But since you merge the objects, it's also possible to leave out the default in the prop (just add a `v-if="project"` to the html, so it will give no errors when the object is null.

Reply:
@Simsteve7 Started this evening, will finish up suggested changes tomorrow.